parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -195,7 +195,15 @@ installCNI() {
     if [[ ! -f "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" ]]; then
         downloadCNI
     fi
-    tar -xzf "$CNI_DOWNLOADS_DIR/${CNI_TGZ_TMP}" -C $CNI_BIN_DIR
+
+    # looking for the folder to do a mv and still fall back to tar method if no folder exists
+    CNI_DIR_TMP=${CNI_TGZ_TMP%.tgz} # Use bash building % to remove the .tgz to look for a folder rather than tgz
+    if [[ -f "$CNI_DOWNLOADS_DIR/${CNI_DIR_TMP}" ]]; then
+        mv ${CNI_DOWNLOADS_DIR}/${CNI_DIR_TMP}/* $CNI_BIN_DIR
```

Comment:
this doesn't match the path in VHD? ${CNI_DOWNLOADS_DIR}/${CNI_DIR_TMP} vs CNI_BIN_DIR?

Reply:
path in VHD should go to downloads. CSE deletes downloads so want to keep everything in there until the final mv or tar cse will do depending on what it finds from vhd